lib/fastlane/plugin/wpmreleasetoolkit/actions/ios/ios_build_prechecks.rb

```diff
@@ -18,7 +18,7 @@ def self.run(params)
         end
 
         # Check local repo status
-        other_action.ensure_git_status_clean()
+        other_action.ensure_git_status_clean() unless params[:run_on_ci]
```

Comment:
Rather than declaring an action parameter for this, could we use detect it using https://docs.fastlane.tools/actions/is_ci/?

Reply:
This makes sense!
Updated!